### DISCUSSION TITLE
Staking stole my money on clearpool.finance

### DISCUSSION THREAD

**Discussion by Ocean47** [clearpool-finance/community#22] [Bug reports]
I tried staking with my trustwallet, Connected my wallet to the site, and tried to stake the wintermute oracle pool, and it stole my tokens, and now shows I haven't staked anything??? what the heck???

**Comment by georgeciubotaru** [comment]
Hey @Ocean47 sorry to hear that. 
Your money is safe, you can even check that using Etherscan Explorer.

The issue will be fixed as soon as possible.

Could you give it a try in around 30 minutes?

Thank you for the report!

**Comment by Ocean47** [comment]
Thanks, that's great to hear.  I'll check again in a little bit.  Again
thanks for your help, and quick response!!!  Much appreciated...

Clint

**Comment by georgeciubotaru** [comment]
Hey, @Ocean47 Please give it a try and feel free to close the issue if all is resolved.

Thank you again for the report!

**Comment by Ocean47** [comment]
Yes, it worked!
Really thank-you. I replied on a couple forums as I wasn't sure where to
reply to before i found your main site. I'll removed/comment each of them
how you helped me. Again, thank-you.  Speaks well of your project. I very
much will add to it I think in the future. Hoping it goes easier next
time.. ;)  But regardless knowing my money is safe as can be expected is a
serious plus.

Thanka again.
Best of luck with your project, i invested in it because you guys seemed to
have it together, glad so far it seems like I was right.

Clint

**Reply by georgeciubotaru** [reply]
Thank you for your interest in using our protocol! We appreciate your support and are thrilled to have you on board.

We encourage you to explore our range of projects, where you can experience the full capabilities of our protocol in action. We're confident it will add value to your goals, and we're excited to see how you use it to create and innovate.

For the latest updates, insights, and announcements, please join us on [Discord](https://discord.gg/YYzxscA4nu) or follow us on [X](https://twitter.com/ClearpoolFin). We look forward to staying connected and keeping you informed about our latest developments.

Warm regards,
George
